content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -5,7 +5,7 @@ description: The PhysX Primitive Collider component adds a PhysX collider to an
 toc: true
 ---
 
-The **PhysX Primitive Collider** component adds a PhysX collider to an entity so that the entity can be included in PhysX simulation. The collider can be defined by a simple shape primitive selected in the the PhysX Primitive Collider component. The PhysX Primitive Collider component can also define a trigger area or a force region.
+The **PhysX Collider** component adds a PhysX collider to an entity so that the entity can be included in PhysX simulation. The collider can be defined by a mesh you create, automatically generated convex meshes, shapes that have been automatically fit to a decomposed mesh, or a simple shape primitive selected in the PhysX Collider component. The PhysX Collider component can also define a trigger area or a force region.
```

Comment:
```suggestion
The **PhysX Primitive Collider** component adds a PhysX collider to an entity so that the entity can be included in PhysX simulation. The collider can be defined by a simple shape primitive selected in the the PhysX Primitive Collider component. The PhysX Primitive Collider component can also define a trigger area or a force region.
```